osh/hub/settings.py

```diff
@@ -56,7 +56,7 @@
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
         'DIRS': [
             # `/etc/osh/hub/templates` overrides `index-content.html`
-            os.path.join("etc", "osh", "hub", "templates"),
+            os.path.join(os.path.sep, "etc", "osh", "hub", "templates"),
```

Comment:
I guess this worked only because `systemd` runs `httpd` with `cwd` set to `/`.  Anyway, I do not think that `os.path.join()` adds here any value.  Cannot we just replace it with `"/etc/osh/hub/templates"`?

It will be much easier to read...